pkg/cmd/pulumi/package_extract_schema.go

```diff
@@ -26,15 +26,17 @@ import (
 func newExtractSchemaCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "get-schema <schema_source>",
```

Comment:
Should this document the new parameters as well?

Reply:
Ah yes good catch